iXBRLViewerPlugin/viewer/src/js/inspector.js

```diff
@@ -531,8 +536,13 @@ Inspector.prototype._selectionSummaryAccordian = function() {
                 );
             }
             this._updateEntityIdentifier(fact, factHTML);
-            this._updateValue(fact.readableValue(), false, factHTML);
-            $('tr.accuracy td', factHTML).text(fact.readableAccuracy());
+            this._updateValue(fact, false, factHTML);
+
+            $('tr.accuracy td', factHTML).empty().append(fact.readableAccuracy());
```

Comment:
Why change from `.text` to `.append`?  Why `.append` here but `.text` in `_updateValue`?

Reply:
Good point.  I briefly toyed with having readableAccuracy return jQuery objects, so switched to append.  Will change it back.

Reply:
Fixed in 4009081